### PR TITLE
Increased test depth

### PR DESCRIPTION
**Pull request by aadams**
It is possible to cross multiple empty words, so we wanted to make tests to reflect that

### CODE REVIEW

File: src/PoolTickBitmap.sol [hunk 1]
```diff
@@ -75,23 +73,13 @@ library PoolTickBitmap {
...
 
-        // TODO: Add a test that searches through multiple word boundaries.
+        // tick next at this point is either 
+        // initialized (never enters this loop) or a word boundry
         while (!initialized && (upper ? tickNext < tickMax : tickNext > tickMax)) {
-            tick = upper
-                ? tick + TICK_WORD_LENGTH * poolVariables.tickSpacing
-                : tick - TICK_WORD_LENGTH * poolVariables.tickSpacing;
-            (tickNext, initialized) = _nextInitializedTickWithinOneWord(poolVariables, tick, !upper);
+            (tickNext, initialized) = _nextInitializedTickWithinOneWord(poolVariables, upper ? tickNext : tickNext - 1, !upper);
```
Comment (snreynolds) [inline src/PoolTickBitmap.sol:82]: 🫠🫠🫠🫠🫠🫠🫠 this looks good

File: test/Depth.t.sol [hunk 2]
```diff
@@ -112,17 +114,25 @@ contract DepthTest is Test {
...
+        // we need the position to not round down (due to FFI constraints)
         vm.assume(delta.liquidityDelta > 1e9);
 
         // make sure we don't overflow liquidity per tick
         vm.assume(delta.liquidityDelta < (pool.maxLiquidityPerTick() / 2));
 
         int24 tickSpacing = pool.tickSpacing();
+
+        // tick in our fuzzer are between -32,768 and 32,767 (int16)
+        // the max depth possible is ~(-1000, 1000), thus thus we truncate
```
Comment (snreynolds) [inline test/Depth.t.sol:126]: "the max depth we are testing"
